utils/strlcpy.h

```diff
@@ -0,0 +1,8 @@
+#ifndef __STRLCPY_H
+#define __STRLCPY_H
+
+#include <sys/types.h>
+
+size_t strlcpy(char *dst, const char *src, size_t size);
```

Comment:
And also let switch all the strncpy to strlcpy at the same time in the following files:
cli/nbd-cli-cmd.c
daemon/gluster.c
daemon/nbd-svc-routines.c

And comment it here for the reason:

For strncpy there is one warning:
Warning: If there is no null byte among the first n bytes of src, the string placed in dest will not be null-terminated. The strlcpy code comes from the BSD and tcmu-runner project.

Reply:
OK